### ISSUE TITLE
Replace @ with try/catch

### ISSUE THREAD

**Issue by spadefoot** [spadefoot/kohana-orm-leap#4]
Don't suppress errors using @, this makes debugging issues almost impossible. Kohana will convert these errors into exceptions and you can handle them using try/catch.

**Comment by taai** [comment]
Well, will it be done? Or I could help?

**Comment by bluesnowman** [comment]
@taai Yes, the plan is still to replace the @.  If you would like to tackle that, that would be great.
